test/irb/test_raise_no_backtrace_exception.rb

```diff
@@ -7,6 +7,7 @@ def test_raise_exception
       bundle_exec = ENV.key?('BUNDLE_GEMFILE') ? ['-rbundler/setup'] : []
       assert_in_out_err(bundle_exec + %w[-rirb -W0 -e IRB.start(__FILE__) -- -f --], <<-IRB, /Exception: foo/, [])
       e = Exception.new("foo")
+      puts e.inspect
```

Comment:
:scissors:

Reply:
That output is what the test is testing for. Without the `puts` the test will fail.